lib/users/api.js

```diff
@@ -636,6 +636,18 @@ class API {
     await requests.delete(uri, this.header_);
   }
 
+    /**
+   * Force kills the job with the given ID, which must be scheduled or queued.
```

Comment:
`Force kills the job with the given ID. Only available for jobs in a queued or scheduled state.